include/SDL3/SDL_gpu.h

```diff
@@ -159,6 +159,24 @@
  * and reasonable to implement across several platforms and underlying APIs.
  * So while these things are not in the "never" category, they are definitely
  * not "near future" items either.
+ *
+ * ## System Requirements
+ *
+ * **Vulkan:** Supported on Windows, Linux, and Nintendo Switch. Requires Vulkan 1.0 with the following extensions and device features:
+ * - `VK_KHR_swapchain`
+ * - `VK_KHR_maintenance1`
+ * - `independentBlend`
+ * - `imageCubeArray`
+ * - `depthClamp`
+ * - `shaderClipDistance`
+ * - `drawIndirectFirstInstance`
+ *
+ * **D3D12:** Supported on Windows 10 or newer, Xbox One (GDK), and Xbox Series X (GDK). Requires a GPU that supports DirectX 12 Feature Level 11_1.
```

Comment:
Teeny change but it turns out they're particular about the naming:

```suggestion
 * **D3D12:** Supported on Windows 10 or newer, Xbox One (GDK), and Xbox Series X|S (GDK). Requires a GPU that supports DirectX 12 Feature Level 11_1.
```